### DISCUSSION TITLE
[RELEASE] Tag der Untoten

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#199] [Map Releases]
Greetings this is my Map that is inspired by the Call of Duty: Cold War "remake" of Nacht der Untoten
This is a pretty decently sized map so it's very easy although due to it being located in
Nacht der Untoten we got no Perks...

> DISCLAIMER!
> This is unfinished and abandoned
(This map sucks due to waypoints not being designed for such a map)

(May run on the PSP or it may walk instead who knows...)

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/10777366/TdU.zip)

**Comment by BCDeshiG** [comment]
You might want to put some screenshots.

**Comment by Paindeluxe** [comment]
Uh so this map sucks I don’t recommend you to play it and don‘t even try to play it on anything other than pc waypoints aren’t great for open maps…

**Comment by MotoLegacy** [comment]
> Uh so this map sucks I don’t recommend you to play it and don‘t even try to play it on anything other than pc waypoints aren’t great for open maps…

zombies won't tread off the path outside of FTE if you're within a certain vicinity. this is easily combated by just having more/higher frequency waypoints and shortening the distance between them. the max amount is 256 so with a smaller map, go absolutely nuts.
